OpenRA.Mods.Cnc/Graphics/Voxel.cs

```diff
@@ -76,7 +80,7 @@ public float[] TransformationMatrix(uint limb, uint frame)
 			return t;
 		}
 
-		public VoxelRenderData RenderData(uint limb)
+		public ModelRenderData RenderData(uint limb)
```

Comment:
Ok, I thought it is preferred to keep interface signature and not rename it based on context. It would simply be:
return limbData[~limb~ section].RenderData;